src/ApiService/ApiService/onefuzzlib/NodeOperations.cs

```diff
@@ -0,0 +1,34 @@
+﻿using ApiService.OneFuzzLib.Orm;
+using Azure.Data.Tables;
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.OneFuzz.Service;
+
+public interface INodeOperations : IOrm<Node>
+{
+    Task<Node?> GetByMachineId(Guid machineId);
+}
+
+public class NodeOperations : Orm<Node>, INodeOperations
+{
+    
+    public NodeOperations(IStorage storage)
+        :base(storage)
+    {
+        
+    }
+
+    public async Task<Node?> GetByMachineId(Guid machineId)
+    {
+        var tableClient = await GetTableClient("Node");
+
+        var data = QueryAsync(filter: $"RowKey eq '{machineId}'");
```

Comment:
How is tableClient used? It looks like you're just passing the filter to QueryAsync, but no actual reference to the table.

Reply:
yes not needed. Removed.